### DISCUSSION TITLE
Odd issue with environment variables on windows

### DISCUSSION THREAD

**Discussion by ohsayan** [community/community#24979] [Actions]
<p>If you take a look at <a href="https://github.com/terrabasedb/terrabasedb/actions/runs/310664952" rel="noopener nofollow ugc">this run</a>, the builds (some steps) for Linux and macOS were skipped, which is expected behavior. However, the same doesn’t happen on Windows. Would setting the default shell to bash fix this…?</p>

**Comment by ohsayan** [comment, marked as answer]
<p>Turns out that setting the default shell to bash fixes this.</p>

**Comment by danielo515** [comment]
<p>This is confusing as hell.<br>
I wasted lots of hours because of this undocumented stupid detail. Is it that hard to add a note on the env-variables docs about setting them on windows?</p>
